apps/web/src/screens/Request/Signature/Approval.tsx

```diff
@@ -0,0 +1,88 @@
+import { type FC } from 'react';
+import { ActivityIndicator } from 'react-native';
+import { AlertCircle } from '@walless/icons';
+import { Button, Image, Stack, Text } from '@walless/ui';
+import { HeaderRequest } from 'components/HeaderRequest';
+import LightText from 'components/LightText';
+
+import { logoSize, logoUri } from '../shared';
+
+interface Props {
+	content?: string;
+	onDeny: () => void;
+	onApprove: () => void;
+}
+
+export const RequestSignatureApproval: FC<Props> = ({
+	content,
+	onDeny,
+	onApprove,
+}) => {
+	return (
+		<Stack flex={1} backgroundColor="#19232C">
```

Comment:
whenever we see Stack/Tamagui, try to kill it, replace with bare react-native

Reply:
these request UI gonna be replaced in final UI updating